commands/entity-store.ts

```diff
@@ -199,22 +205,21 @@ export const cleanEntityStore = async () => {
   }
 };
 
-
-const matchUsersAndHosts = (users, hosts) => {
+const matchUsersAndHosts = <U extends object, H extends object, UA extends { user: U }, HA extends { host: H }>(users: Array<UA>, hosts: Array<HA>): { users: Array<UA&{host?:H;}>, hosts: Array<HA&{user?:U}> } => {
```

Comment:
Good code, right? :) :) :)

Reply:
U = user,
UA = user alert
H = host
H = host alert

I think we should fix this up before merging... maybe

Reply:
I added some interfaces and simplified a little